topi/src/topi.cc

```diff
@@ -206,6 +206,11 @@ TVM_REGISTER_GLOBAL("topi.nn.pad")
   *rv = pad(args[0], args[1], args[2], args[3]);
   });
 
+TVM_REGISTER_GLOBAL("topi.nn.matmul")
```

Comment:
matmul is not nn specific. It should be `topi.matmul`

Reply:
Actually, in C++ matmul defined in nn.h which is strange. I assume I should move it too. Could you suggest new location for matmul? How about `transform.h` / `transform.py` ?

Reply:
I've moved it to transform.h for now, please let me know if it is not OK.